docs/manual/templates.rst

```diff
@@ -119,11 +119,13 @@ in the template. Typically, these vary per form!
 
 These are additional variables and statements available to the template.
 
-===================  ===========================================================================
-Element              Description
-===================  ===========================================================================
-``{% summary %}``    A full summary of all elements marked to show in the email.
-===================  ===========================================================================
+===================================  ===========================================================================
+Element                              Description
+===================================  ===========================================================================
+``{% summary %}``                    A full summary of all elements marked to show in the email.
+``{% appointment_information %}``    The information about the appointment to show in the email.
+``{% get_appointment_links %}``      Retrieves relevant links about the appointment.
```

Comment:
Not sure if we want more documentation on this since this essentially adds information that _can_ be used but doesn't display any information itself which makes it different than `appointment_information` or `summary`.

Reply:
can we also keep it a little consistant, and rename `get_appointment_links` to `appointment_links` in the template?

Reply:
it's meant to be used as `{% get_appointment_links as links %} {{ links.cancel_url|urlize }}` for example, so I think the `get_` is more semantic. In the future, this will also include the `change_url`.